sklearn/svm/classes.py

```diff
@@ -448,20 +448,27 @@ class SVC(BaseSVC):
         Penalty parameter C of the error term.
 
     kernel : string, optional (default='rbf')
-         Specifies the kernel type to be used in the algorithm.
-         It must be one of 'linear', 'poly', 'rbf', 'sigmoid', 'precomputed' or
-         a callable.
-         If none is given, 'rbf' will be used. If a callable is given it is
-         used to pre-compute the kernel matrix from data matrices; that matrix
-         should be an array of shape ``(n_samples, n_samples)``.
+        Specifies the kernel type to be used in the algorithm.
+        It must be one of 'linear', 'poly', 'rbf', 'sigmoid', 'precomputed' or
+        a callable.
+        If none is given, 'rbf' will be used. If a callable is given it is
+        used to pre-compute the kernel matrix from data matrices; that matrix
+        should be an array of shape ``(n_samples, n_samples)``.
 
     degree : int, optional (default=3)
         Degree of the polynomial kernel function ('poly').
         Ignored by all other kernels.
 
     gamma : float, optional (default='auto')
         Kernel coefficient for 'rbf', 'poly' and 'sigmoid'.
-        If gamma is 'auto' then 1/n_features will be used instead.
+        If gamma is 'auto' then 1/n_features will be used.
+        If gamma is 'scale' then 1/(n_features * X.std()) will be used.
+        The current default 'auto' is deprecated in version 0.20 and will
```

Comment:
I guess the formulation here was what led me to think that ``'auto'`` is deprecated, i.e. will be removed. I would rather say that "The default value will change to 'scale' in 0.22".  I'm not sure about a good way to say when this decision was made. I think we had "This warning was introduced in version X" somewhere.